node/src/test_utils/mod.rs

```diff
@@ -182,7 +182,7 @@ pub fn make_meaningless_message_type() -> MessageType {
     DnsResolveFailure_0v1::new(make_meaningless_stream_key()).into()
 }
 
-pub fn make_route(public_keys: Vec<&PublicKey>) -> Route {
+pub fn make_route_to_proxy_client(public_keys: Vec<&PublicKey>) -> Route {
```

Comment:
You could include `one_way_route` in the name. It's not a requirement though.

Reply:
Done :+1: